app/views/group/educations/_actions_index.html.haml

```diff
@@ -0,0 +1,6 @@
+-#  Copyright (c) 2023  , Pfadibewegung Schweiz. This file is part of
+-#  hitobito_youth and licensed under the Affero General Public License version 3
+-#  or later. See the COPYING file at the top-level directory or at
+-#  https://github.com/hitobito/hitobito_youth.
+
+= action_button(t('.export_people'), educations_path({format: :csv}.merge(params.to_unsafe_h)), :download)
```

Comment:
You added the capability for CSV and XLSX to the controller, but only expose CSV here. Wouldn't XLSX be more valuable? Maybe in the future we might want to color some of the text in the spreadsheet, depending on when some qualifications expire.

The CSV is clearly easier to test though.